docs/reference/release-notes/release-notes-2_0_0.asciidoc

```diff
@@ -0,0 +1,143 @@
+
+[float]
+=== Deprecations
+
+Mapping:
+
+* Deprecate index.analysis.analyzer.default_index in favor of index.analysis.analyzer.default. {pr}/14027[#14027] (issue: {issue}/11861[#11861])
+
+[float]
+=== Enhancements
+
+Core:
+
+* Make PerThreadIDAndVersionLookup per-segment {pr}/14070[#14070]
+
+Internal:
+
+* TransportNodesAction shouldn't hold on to cluster state {pr}/13948[#13948]
+
+Packaging:
+
+* Packaging: change permissions/ownership of config dir {pr}/14017[#14017] (issue: {issue}/11016[#11016])
+* Release: Fix package repo path to only consist of major version {pr}/13971[#13971] (issue: {issue}/12493[#12493])
+
+Plugin Cloud GCE:
+* Add _gce_ network host setting {pr}/13612[#13612] (issues: {issue}/13590[#13590], {issue}/13605[#13605])
+
+Plugins:
+
+* Check "plugin already installed" before jar hell check. {pr}/14207[#14207] (issue: {issue}/14205[#14205])
+* Plugin script to set proper plugin bin dir attributes {pr}/14088[#14088] (issue: {issue}/11016[#11016])
+* Plugin script to set proper plugin config dir attributes {pr}/14048[#14048] (issue: {issue}/11016[#11016])
+
+[float]
+=== Bug Fixes
+
+CRUD:
+
+* Send response for update request when it timed out {pr}/13101[#13101]
+
+Engine:
+
+* Never wrap searcher for internal engine operations {pr}/14071[#14071]
+Fielddata:
+* Add GeoPoint in StreamInput/StreamOutput {pr}/13632[#13632] (issue: {issue}/13340[#13340])
+
+Internal:
+
+* Properly bind ClassSet extensions as singletons {pr}/14232[#14232] (issue: {issue}/14194[#14194])
+* Streamline top level reader close listeners and forbid general usage {pr}/14084[#14084]
+
+Logging:
+
+* Settings in log config file should not overwrite custom parameters {pr}/13934[#13934]
+* Plugin cli tool should not create empty log files {pr}/13933[#13933]
+
+Mapping:
+
+* Enforce metadata fields are not passed in documents {pr}/14003[#14003] (issues: {issue}/11074[#11074], {issue}/13740[#13740])
+Plugin Cloud AWS:
+* Discovery-ec2 plugin should check discovery.type {pr}/13814[#13814] (issues: {issue}/13581[#13581], {issue}/13809[#13809])
+
+Plugins:
+
+* Parse Java system properties in plugin.bat {pr}/13989[#13989] (issue: {issue}/13616[#13616])
+* PluginManager: Dont leave leftover files on unsuccessful installs #12851 (issue: {issue}/12749[#12749])
+
+REST:
+
+* Restore support for escaped '/' as part of document {pr}/14216[#14216] (issues: {issue}/13665[#13665], {issue}/13691[#13691], {issue}/14177[#14177])
+* Add missing REST spec for detect_noop {pr}/14004[#14004]
+* Expose nodes operation timeout in REST API {pr}/13981[#13981]
+* Ensure XContent is consistent across platforms {pr}/13816[#13816]
+
+Shadow Replicas:
+
+* Don't pull translog from shadow engine {pr}/14000[#14000] (issue: {issue}/12730[#12730])
+Snapshot/Restore:
+* Snapshot restore operations throttle more than specified {pr}/13828[#13828] (issue: {issue}/6018[#6018])
+
+[float]
+=== Upgrades
+
+Network:
+
+* Upgrade Netty to 3.10.5.final {pr}/14105[#14105]
+
+[float]
+=== Build
```

Comment:
For the release notes, I exclude everything from Build onwards (ie Build, Test, Docs, Not classified etc)